src/dsp_tools/utils/excel2json/properties.py

```diff
@@ -293,7 +293,7 @@ def _check_missing_values_in_row_raise_error(df: pd.DataFrame, excelfile: str) -
     if missing_dict:
         # Get the row numbers from the boolean series
         missing_dict = utl.get_wrong_row_numbers(wrong_row_dict=missing_dict, true_remains=True)
-        error_str = "\n".join([f" - Column Name: {k} Row Number: {v}" for k, v in missing_dict.items()])
+        error_str = "\n".join([f"- Column Name: {k} Row Number: {v}" for k, v in missing_dict.items()])
```

Comment:
```suggestion
        error_str = "\n".join([f"- Column '{k}', Row Number(s) {v}" for k, v in missing_dict.items()])
```

This would produce a much nicer output:
```
The file 'Test' is missing values in the following rows:
- Column 'name', Row Number(s) [9]
- Column 'super', Row Number(s) [2, 4, 9]
```

If I look at you output below, I find it hard to read